src/components/navbar/navbar.css

```diff
@@ -3,11 +3,25 @@
   width: 100%;
   box-shadow: var(--nav-bar-shadow);
   grid-area: nav;
-  z-index: 1005;
+  z-index: 2;
   position: sticky;
   top: 0;
   background-color: var(--background-color);
 }
+.nav-search-bar {
+  border: 2px solid var(--secondary-color);
+  border-radius: 4.8px;
```

Comment:
you should use rounded numbers  as decimals do no apply on pixels

Reply:
done